TypeCobol/Compiler/SqlScanner/SqlScanner.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.Scanner;
+
+namespace TypeCobol.Compiler.SqlScanner
+{
+    /// <summary>
+    /// A Sql Scanner related to a Scanner 
+    /// </summary>
+    public class SqlScanner
+    {
+        public static bool IsSqlKeyword(ref int currentIndex, int startIndex, int lastIndex, string line, TokensLine tokensLine, out Token sqlToken)
```

Comment:
Since it actually reads the chars and change the current index, this should be refactored as `public static Token ScanSqlKeywordOrExecStatementText(string line, ref int currentIndex, int startIndex, int lastIndex, TokensLine tokensLine)` combining the logic here and what is done in Scanner.cs where this method is called.

Reply:
done in 5a26682